### PR TITLE
fix thrown errors, enable js strict mode

### PR DESCRIPTION
**Pull request by hdert**
sorry about the number of changes in the diff, the code goes through the beautifier on save.

### CODE REVIEW

**Comment by hdert** [comment]
Closes #11

File: js/main.js [hunk 1]
```diff
@@ -95,10 +98,10 @@ async function closeSkill() {
     document.getElementById("skill-title").textContent = "";
     document.getElementById("skill-desc").innerHTML = "";
 }
-  
+
 // Randomly load an image to be used for the 404 meme
 function load404Image() {
-    images = [
+    var images = [
```
Comment (hdert) [inline js/main.js:104]: This stops an error message and makes images a local variable instead of a global variable

File: js/main.js [hunk 2]
```diff
@@ -107,7 +110,7 @@ function load404Image() {
         "incomplete_archives.jpg"
     ];
 
-    imgElement = document.getElementById("404-img");
+    var imgElement = document.getElementById("404-img");
```
Comment (hdert) [inline js/main.js:113]: Same here

**Review by JHay0112 — APPROVED** [review]
Someone needed to fix my terrible JS, thank you.

**Pull request merged**